apps/account/templates/account/detail.html

```
@@ -1,217 +1,217 @@
+                    <span class="badge text-bg-secondary">
+                      {% if user.paypal_me_username %}
+                        {% trans "PayPal linked" %}
+                      {% else %}
+                        {% trans "No PayPal link" %}
+                      {% endif %}
+                    </span>
+                  </div>
                 </div>
+                {% if request.user.id == user.id %}
+                  <div class="ms-auto">
+                    <button class="btn btn-primary d-flex align-items-center gap-2"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)